### PR TITLE
Update cross-tenant-onedrive-migration.md

### LINKED ISSUE #11827: Ambiguity on the information provided. Or a Typo
Ambiguity in the documentation. This might be due to a Typo. Under the size limit, there is a 2TB content max, but in the "Important" section, it says 2GB. 

Page: https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-onedrive-migration?view=o365-worldwide#onedrive-account-size-limits

Thanks.

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the limit.

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11827

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/cross-tenant-onedrive-migration.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Any legal URL will be accepted when creating your Identity Map from Source to Ta
 Each OneDrive account can have a maximum of 2 TB of content or 1 million items.
 
 >[!Important]
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-onedrive-migration.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/enterprise/cross-tenant-onedrive-migration.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Any legal URL will be accepted when creating your Identity Map from Source to Ta
...
->If you attempt to migrate any OneDrive site that exceeds the 2GB quota, the transfer will fail.
+>If you attempt to migrate any OneDrive site that exceeds the 2TB quota, the transfer will fail.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-onedrive-migration.md]:
```suggestion
> If you attempt to migrate any OneDrive site that exceeds the 2 TB quota, the transfer will fail.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoanneHendrickson Please merge this PR. Thanks

**Pull request merged**